### DISCUSSION TITLE
How do I test for available RAM to number of cores ratio from portfile.cmake

### DISCUSSION THREAD

**Discussion by Matioupi** [microsoft/vcpkg#17911] [Ideas]
Hello,

I believe this would be nice to be able to test the available RAM amount and number of host cores from within the portfile.cmake.

One possible reason would be to decide using or not using the DISABLE_PARALLEL when building arm libraries on some SBC where the limited amount of RAM results in failed compilation is not disabling parallel builds.

Are there any existing ways to emulate this right now, like setting DISABLE_PARALLEL if RAM to number of core ratio is below a given threshold ?

**Comment by BillyONeal** [comment]
We don't have such a feature at this time.
